src/browser/documentation/guides/movie-graph.jsx

```diff
@@ -43,6 +47,30 @@ const slides = [
         <li>Query: discover related actors and directors</li>
         <li>Solve: the Bacon Path</li>
       </ol>
+      <p></p>
+      <p>
+        WARNING: This guide will modify the data in the currently active
+        database.{' '}
+      </p>
+      <div>
+        If you don't want to modify your data:
+        <ol>
+          <li>
+            Switch to system database{' '}
+            <TextCommand command={'use system'}></TextCommand>
+          </li>
+          <li>
+            Create a new databse{' '}
+            <pre
+              mode="cypher"
+              className="pre-scrollable code runnable"
+              style={{ width: 'fit-content' }}
+            >
```

Comment:
@shkirando  It'd be nice if the commands were runnable easily. So I'm thinking we could have it be: `create database movies` and then `:use movies`. As steps 2 and 3